Tests/Service/FeatureServiceTest.php

```diff
@@ -64,28 +99,27 @@ public function testDefaultVoter()
     }
 
     /**
-     * @return VoterRegistry
+     * Test new feature
      */
-    protected function getRegistry()
+    public function testFeatureUnknownedVoter(): void
     {
+        $this->expectException(VoterNotFoundException::class);
 
-        // Create alwaysTrueVoter stub
-        $alwaysTrueVoter = $this->getMock('\Ecn\FeatureToggleBundle\Voters\VoterInterface');
-        $alwaysTrueVoter->expects($this->any())
-            ->method('pass')
-            ->will($this->returnValue(true));
-
-        // Create alwaysFalseVoter stub
-        $alwaysFalseVoter = $this->getMock('\Ecn\FeatureToggleBundle\Voters\VoterInterface');
-        $alwaysFalseVoter->expects($this->any())
-            ->method('pass')
-            ->will($this->returnValue(false));
+        // Define a feature
+        $features = [
+            'testfeature' => [
+                'voter' => 'testVoter',
+                'params' => []
+            ]
+        ];
+        $default = [
+            'voter' => null,
+            'params' => []
+        ];
 
-        // Create a registry with the voter stubs in it
-        $registry = new VoterRegistry();
-        $registry->addVoter($alwaysTrueVoter, 'AlwaysTrueVoter');
-        $registry->addVoter($alwaysFalseVoter, 'AlwaysFalseVoter');
+        // Create service
+        $service = new FeatureService($features, $default, $this->getRegistry());
 
-        return $registry;
+        $service->has('testfeature');
```

Comment:
Missing assert!

Reply:
@elbcoast 
here the assertion will not be achieved because there are an exception (which we are testing here)
$this->assertTrue($service->has('testfeature'));